web/src/components/new/TagCreateDialog.vue

```diff
@@ -7,6 +7,20 @@
         </v-card-title>
         <v-card-text>
           <v-text-field v-model="tagName" label="Name" autofocus></v-text-field>
+          <v-text-field v-model="tagColor" label="Color" hide-details>
+            <template v-slot:append>
+              <v-menu v-model="tagColorPickerOpen" top nudge-bottom="270" nudge-left="32" :close-on-content-click="false">
+                <template v-slot:activator="{ on }">
+                  <div :style="swatchStyle" v-on="on" />
+                </template>
+                <v-card>
+                  <v-card-text>
+                    <v-color-picker v-model="tagColor" mode="hexa" hide-mode-switch show-swatches flat />
```

Comment:
i'm not 100% happy with this, i played around with the color picker and found a mode where only the selected color and a single slider is visible, i think that would be the most useful widget to show instead of the textbox.
the only issue is that one probably wants to show the full dialog when the user really wants, would be nice if one could open that dialog with a click on the color, but the color picker widget doesn't seem to be very flexible :/